mbta.js

```diff
@@ -0,0 +1,89 @@
+const subway_lines = {
+    Red : [
+    'South Station',
+    'Park Street',
+    'Kendall',
+    'Central',
+    'Harvard',
+    'porter',
+    'Davis',
+    'Alewife'
+    ],
+    
+    Green : [    
+    'Government Center',
+    'Park Street',
+    'Boylston',
+    'Arlington',
+    'Copley',
+    'Hynes',
+    'Kenmore'
+    ],
+    Orange : [
+    'North Station',
+    'Haymarket',
+    'Park Street',
+    'State',
+    'Downtown Crossing',
+    'Chinatown',
+    'Back Bay',
+    'Forest Hills'
+    ]
+    }
+    
+    const stopsBetweenStations = function (startLine, startStation, endLine, endStation)
+     {
+         let countOfStops = 0;
+         const startLine_index = subway_lines[startLine].indexOf(startStation)
+         const endLine_index = subway_lines[endLine].indexOf(endStation)
+    
+    
+         if (startLine === endLine)
+         {
+           
+         return countOfStops ;
```

Comment:
this causes an error, if to stops are in the same line, but away from each other always will return 0.